plugins/rule_engines/irods_rule_language/src/functions.cpp

```diff
@@ -1907,7 +1907,12 @@ Res *smsi_remoteExec( Node** paramsr, int, Node* node, ruleExecInfo_t* rei, int,
 
     try {
         auto taggedValues = getTaggedValues(params[1]->text);
-        auto it = taggedValues.find("ZONE");
+        auto it = taggedValues.find("INST_NAME");
+        if ( it != taggedValues.end() ) {
+            addKeyVal( &execMyRuleInp.condInput, "instance_name", it->second.front().c_str());
```

Comment:
"instance_name" -> `irods::KW_CFG_INSTANCE_NAME`

Reply:
noticed that, will use the symbol.

Reply:
hm, there are several keywords that would all seem to fit, but the simplest seems to be INSTANCE_NAME_KW just as it is here, in a function called from _delayExec: https://github.com/irods/irods/blob/fc0653187ec7a308a466529aa76917c723c06995/server/re/src/irods_re_structs.cpp#L519

Reply:
Oh, I didn't know that existed, or that we had multiple of that keyword. I think what you used is better, personally. Let's go with `INSTANCE_NAME_KW`.